### PR TITLE
Add BLAST token to tokens_blast_erc20.sql

### PR DESCRIPTION
**Pull request by akincibor**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:**

- Add BLAST token to tokens_blast_erc20.sql

---

### CODE REVIEW

**Comment by akincibor** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jeff-dude** [comment]
this token already exists
```
select *
from tokens.erc20
where blockchain = 'blast'
and contract_address = 0xb1a5700fA2358173Fe465e6eA4Ff52E36e88E2ad
```

File: dbt_subprojects/tokens/models/tokens/blast/tokens_blast_erc20.sql [hunk 1]
```diff
@@ -14,6 +14,7 @@ SELECT
 FROM (VALUES
     (0x4300000000000000000000000000000000000004, 'WETH', 18)
     , (0x4300000000000000000000000000000000000003, 'USDB', 18)
+    , (0xb1a5700fA2358173Fe465e6eA4Ff52E36e88E2ad, 'BLAST', 18)
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/tokens/blast/tokens_blast_erc20.sql:17]: https://blastscan.io/token/0xb1a5700fA2358173Fe465e6eA4Ff52E36e88E2ad

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @akincibor

**Pull request closed without merging**